index.html

```diff
@@ -82,37 +88,36 @@ <h2 id="modalTitle">Add New Item</h2>
     </div>
     </div>
 
+    <!-- FILTERS AND OPTIONS SIDEBAR -->
+     <div id="optionsModal" class="optionsModal" role="dialog" aria-labelledby="optionsModalTitle" aria-modal="true" >
```

Comment:
I *think* using a modal is okay here - and because it's dynamic the `dialog` role should ensure that when the settings panel opens, this is announced to screen readers. 

I should do more investigation though.